plugins/awards-backend/src/awards.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright SeatGeek
+ * Licensed under the terms of the Apache-2.0 license. See LICENSE file in project root for terms.
+ */
+import { NotFoundError } from '@backstage/errors';
+import { Award, AwardInput } from '@seatgeek/backstage-plugin-awards-common';
+import { Logger } from 'winston';
+import { AwardsStore } from './database/awards';
+
+export class Awards {
+  private readonly db: AwardsStore;
+  private readonly logger: Logger;
+
+  constructor(db: AwardsStore, logger: Logger) {
+    this.db = db;
+    this.logger = logger.child({ class: 'Awards' });
+    this.logger.debug('Constructed');
```

Comment:
is the logger here context aware? otherwise would remove or make a little more descriptive

Reply:
yep it becomes context aware (ish) when we do `logger.child` on the line above